tegra-fuse.c

```diff
@@ -170,10 +188,13 @@ tegra_fuse_context_open (const char *basepath)
 	ssize_t typelen;
 	tegra_soctype_t soc;
 	int fd;
-	unsigned long chipid;
+	unsigned short chipid;
```

Comment:
No reason to make this short, is there?

Reply:
I was thinking about `chip_uid` vs `chip_id` 
`chip_uid` is an unsigned long but the `chip_id` can have only the following values:

```
TEGRA210_INT_CID --> 5
TEGRA186_INT_CID --> 6
TEGRA194_INT_CID --> 7
TEGRA234_INT_CID --> 8
``` 

example: 

* Tegra194

```
$ cat /sys/module/tegra_fuse/parameters/tegra_chip_uid 
8171556300411110919
$ # chip_id = (chip_uid >> 60) 
$ echo $((8171556300411110919>>60))
7
```